src/Microsoft.Extensions.Logging/LoggerBuilder.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Logging
+{
+    internal class LoggerBuilder : ILoggerBuilder
```

Comment:
If we want to preserve the ability to new this up without an explicit service collection then it should be public and have a parameters less ctor that news up a service collection internally. But meh.